libs/designsystem/src/lib/components/form-field/input/input.component.scss

```
@@ -1,7 +1,14 @@
 @use '../form-field-inputs.shared' as shared;
+@use "~@kirbydesign/core/src/scss/interaction-state";
 @use "~@kirbydesign/core/src/scss/utils";
 
 :host {
+  @include interaction-state.transition(background-color);
+  @include interaction-state.apply-hover($loudness: 's') {
+    background-color: interaction-state.get-state-color('white');
```

The `get-state-color()` helper/utility function accepts loudness as argument:

```suggestion
    background-color: interaction-state.get-state-color('white', $loudness: 's');
```

or just

```suggestion
    background-color: interaction-state.get-state-color('white', 's');
```